### PR TITLE
Fix: Changed to SentenceCase for DividerComponent

### PR DESCRIPTION
**Pull request by deanurag**
Fixes : https://github.com/ToolJet/ToolJet/issues/7775

I have changed Divide Color to Divide color.

### CODE REVIEW

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -4647,7 +4647,7 @@ export const widgets = [
     styles: {
       dividerColor: {
         type: 'color',
-        displayName: 'Divider Color',
+        displayName: 'Divider color',
```
Comment (tejakummarikuntla) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:4650]: Hey @deanurag, please revert this change.

**Review by tejakummarikuntla — CHANGES_REQUESTED** [review]
Hey @deanurag, could you address the suggested changes?

**Comment by deanurag** [comment]
Yes, I wil be look into them ASAP.

File: frontend/src/Editor/WidgetManager/widgetConfig.js (hunk 1 above)
Comment (deanurag) [reply frontend/src/Editor/WidgetManager/widgetConfig.js]: I have reverted the asked change.

Comment (deanurag) [reply frontend/src/Editor/WidgetManager/widgetConfig.js]: I have reverted the asked change.

**Review by karan-rathod-316 — APPROVED** [review]
LGTM

**Review by tejakummarikuntla — APPROVED** [review]
